src/metabase/models/params.clj

```diff
@@ -111,16 +110,14 @@
   "Parse a Card parameter `target` form, which looks something like `[:dimension [:field-id 100]]`, and return the Field
```

Comment:
On line 110, why isn't the return type `mbql.s/field` instead of `mbql.s/Field`? Given it can't be an expression

Reply:
it could be an expression, see https://github.com/metabase/metabase/pull/34262